packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -0,0 +1,3027 @@
+```
+
+##### Response example
+
+```json
+{
+  "resources": [
+    {
+      "id": "ire106sQKoHoXXsAe0g4",
+      "_links": {
+        "self": {
+          "href": "https://${yourOktaDomain}/api/v1/groups/00guaxWZ0AOa5NFAj0g3"
+        }
+      }
+    },
+    {
+      "id": "ire106riDrTYl4qA70g4",
+      "_links": {
+        "self": {
+          "href": "https://${yourOktaDomain}/api/v1/groups/00gu67DU2qNCjNZYO0g3/users"
+        }
+      }
+    },
+    {
+      "id": "irezvo4AwE2ngpMw40g3",
+      "_links": {
+        "self": {
+          "href": "https://${yourOktaDomain}/api/v1/users"
+        }
+      }
+    },
+    {
+      "id": "irezvn1ZZxLSIBM2J0g3",
+      "_links": {
+        "self": {
+          "href": "https://${yourOktaDomain}/api/v1/groups"
+        }
+      }
+    }
+  ],
+  "_links": {
+    "next": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/resources?after=irezvn1ZZxLSIBM2J0g3"
+    },
+    "resource-set": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g"
+    }
+  }
+}
+```
+
+#### Delete a resource
+<ApiLifecycle access="beta" />
+
+<ApiOperation method="delete" url="/api/v1/iam/resource-sets/${resourceSetId}/resources/${resourceId}" />
+
+Remove a resource from a Resource Set
+
+##### Request parameters
+
+| Parameter      | Description                             | Param Type   | DataType     | Required |
+| :------------- | :-------------------------------------- | :----------- | :----------- | :------- |
+| resourceSetId  | id of the Resource Set                  | URL          | String       | TRUE     |
+| resourceId     | id of the resource within Resource Set  | URL          | String       | TRUE     |
+
+`resourceId` is the id obtained when [listing resources within Resource Set](#list-resources). For example, if the resource object was:
+```json
+    {
+      "id": "ire106sQKoHoXXsAe0g4",
+      "_links": {
+        "self": {
+          "href": "https://${yourOktaDomain}/api/v1/groups/00guaxWZ0AOa5NFAj0g3"
+        }
+      }
+    }
+```
+`ire106sQKoHoXXsAe0g4` could be used as `resourceId` to remove the Groups from list of resources in the set.
+
+##### Response parameters
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+##### Request example
+
+```bash
+curl -v -X DELETE \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/resources/ire106sQKoHoXXsAe0g4"
+```
+
+##### Response example
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+## Custom role assignment operations
+<ApiLifecycle access="beta" />
+These operations allow the assignment and unassignment of Custom Roles. This is done by creating a Binding.
+A Binding is represents an association of a principal, Role and Resource Set uniquely identified by the `bindingId`.
```

Comment:
A Binding is represents... > A Binding represents...
uniquely idendified > that is uniquely identified